alvr/dashboard/src/dashboard/components/settings_controls/mod.rs

```diff
@@ -52,10 +51,14 @@ pub fn get_display_name(id: &str, strings: &HashMap<String, String>) -> String {
     })
 }
 
+pub fn f64_eq(f1: f64, f2: f64) -> bool {
+    format!("{:.6}", f1) == format!("{:.6}", f2)
+}
+
 pub fn serde_values_eq(a: &serde_json::Value, b: &serde_json::Value) -> bool {
     if let (serde_json::Value::Number(n1), serde_json::Value::Number(n2)) = (a, b) {
         if let (Some(f1), Some(f2)) = (n1.as_f64(), n2.as_f64()) {
-            return approx_eq!(f64, f1, f2, epsilon = 1e-6);
+            return f64_eq(f1, f2);
```

Comment:
I still think this should only kick in when it's actually a float, integers should always be handled with a direct comparison. (This avoids issues with large numbers getting rounded to a float with `e6` or something). Also maybe 6 is too many and it gets into the range of messing up on large error, so maybe we should be using 5 or 4?